site/docs/components/cookie_compliance.mdx

```diff
@@ -0,0 +1,136 @@
+
+**Mandatory cookies** (visually grayed-out / disabled)
+
+- Login cookies: A mandatory cookie that allows the visitor to visit the site.
+- Language cookies: We store the interface language you have selected.
+
+**Optional cookies**
+
+- Statistic cookies: The information collected by statistical cookies is used to develop the website further.
+- Preference cookies: Preference cookies customize E.g. the look and feel of a site based on a user's choices.
+- Marketing cookies: Marketing cookies can be applied to target specific content to users.
+
+#### 3. Consent confirmation buttons
+- Accept all cookies
+- Only mandatory
+- Continue with selected (TBD)
+- Opt out (TBD)
+- Close (cross icon) -button
+
+## Usage
+
+### When to use
+ - When you need to inform users about cookie usage. 
+
+### When not to use
+ - For any other usage except informing about cookie usage.
+
+### When to use each banner type?
+| Site uses cookies              | Cookie type                    | What to do and how to inform the user?                                                                              | Example
+| ------------------------------ | ------------------------------ | ------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------- |
+| Yes                            | Only mandatory cookies         | Cookie information needs to be conveyed to the user. Only requires acknowledgment and no choices from the user.     | Acknowlege -button.
+|                                | Only optional cookies          | User is given the option to choose individual cookies or accept all cookies.                                        | Cookie description(s) + duration(s) + select cookies or accept all -button.
+|                                | Mandatory and optional cookies | User is given the option to choose individual cookies, accept all cookies or only mandatory cookies.                | Cookie description(s) + duration(s) + select cookies or accept only mandatory -button.
+| No                             | –                              | **No action needed**.                                                                                               | -
+
+
+### Checklist
+ - Make sure that cookie consent is the first item in the page tab order.
+ - If your page has a chat button in lower right corner, keep the chat button hidden until cookie consent has been accepted and removed from the DOM.
+ - Never use an overlay behind the banner or otherwise restrict use of the site.
+ - Be sure to add a description about the cookie that stores information about accepted cookies.
+
+ **For more information about cookie policies, please visit** <Link href="https://www.traficom.fi/fi/saadokset/evasteohjeistus-palveluntarjoajille" external>Traficom guidelines for service providers.</Link>
+
+## Variations
```

Comment:
Order; this should be placed below the "Security" and above the "Demos & API" section.